File: 프로그래머스/1/76501. 음양 더하기/음양 더하기.py
def solution(absolutes, signs):
    total = 0
    for i in range(len(absolutes)):
        if signs[i]:
            total += absolutes[i] # signs[i]가 True인 경우 양수로 더함
        else:
            total -= absolutes[i] # signs[i]가 False인 경우 음수로 뺌
    return total

File: 프로그래머스/1/76501. 음양 더하기/음양 더하기.js
function solution(absolutes, signs) {
    let sum = 0;
    for (let i = 0; i < absolutes.length; i++) {
        if (signs[i]) {
            sum += absolutes[i]; // signs[i]가 true인 경우 양수로 더함
        } else {
            sum -= absolutes[i]; // signs[i]가 false인 경우 음수로 뺌
        }
    }
    return sum;
}